File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    # 1. 문자열을 소문자로 변환
    lower_case_string = s.lower()
    
    # 2. 'p'와 'y'의 개수를 셈
    p_count = lower_case_string.count('p')
    y_count = lower_case_string.count('y')
    
    # 3. 개수를 비교하여 결과 반환
    return p_count == y_count

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s) {
    // 1. 문자열을 소문자로 변환
    let lowerCaseString = s.toLowerCase();
    
    // 2. 'p'와 'y'의 개수를 셈
    let pCount = 0;
    let yCount = 0;
    
    for (let char of lowerCaseString) {
        if (char === 'p') {
            pCount++;
        } else if (char === 'y') {
            yCount++;
        }
    }
    
    // 3. 개수를 비교하여 결과 반환
    return pCount === yCount;
}